packages/backend/src/errors/retriable-error.ts

```diff
@@ -0,0 +1,18 @@
+import BaseError from './base'
+
+interface RetriableErrorParams {
```

Comment:
nit: suggest converting the interface to type instead so that it is not declared globally and cannot be amended elsewhere